frontend/src/pages/home/Home.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { Button, Hint, HintBody, HintTitle, PageSection } from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
+import { useNavigate } from 'react-router-dom';
+import { useAccessReview } from '~/api';
+import { useBrowserStorage } from '~/components/browserStorage';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import ProjectsSection from './ProjectsSection';
+import OrganizeSection from './OrganizeSection';
+import QuickStartsSection from './QuickStartsSection';
+import EnableTeamSection from './EnableTeamSection';
+
+const accessReviewResource: AccessReviewResourceAttributes = {
+  group: 'project.openshift.io',
+  resource: 'projectrequests',
+  verb: 'create',
+};
+
+const Home: React.FC = () => {
+  const navigate = useNavigate();
+  const [hintHidden, setHintHidden] = useBrowserStorage<boolean>(
+    'odh.dashboard.landing.hint',
+    false,
+    false,
+    false,
+  );
+  const [allowCreate, rbacLoaded] = useAccessReview(accessReviewResource);
+
+  return (
+    <>
+      {!hintHidden ? (
```

Comment:
can we move this whole hint block into another component? It has a lot of text and nuance that really doesn't belong to the Home page component.